warp_drive/training/trainer.py

```diff
@@ -249,6 +219,88 @@ def generate_rollout(self, batch_index):
         torch.cuda.synchronize()
         self.perf_stats.env_step_time += start_event.elapsed_time(end_event) / 1000
 
+    def evaluate_policies(self, batch_index):
+        probabilities = {}
+        for policy in self.policies:
+            probabilities[policy], _ = self.models[policy](
+                batch_index=batch_index, batch_size=self.training_batch_size_per_env
+            )
+            # Combine probabilities if there are multiple policies,
+            # yet they share the same placeholders
```

Comment:
the final probabilities should concatenate in terms of the agent dim (i.e., policies) since we do not have policy placeholder in this case, like [[num_envs, num_agents, action_dim],[...]]